src/main/java/com/example/bookstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,27 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.dto.user.UserRequestDto;
+import com.example.bookstore.dto.user.UserResponseDto;
+import com.example.bookstore.exception.RegistrationException;
+import com.example.bookstore.service.UserService;
+import jakarta.validation.Valid;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/auth")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

add `server.servlet.context-path=/api` to application.properties